lib/Google/Cse.php

```diff
@@ -1,387 +0,0 @@
-<?php
-declare(strict_types=1);
-
-/**
- * Pimcore
- *
- * This source file is available under two different licenses:
- * - GNU General Public License version 3 (GPLv3)
- * - Pimcore Commercial License (PCL)
- * Full copyright and license information is available in
- * LICENSE.md which is distributed with this source code.
- *
- *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
- *  @license    http://www.pimcore.org/license     GPLv3 and PCL
- */
-
-namespace Pimcore\Google;
-
-use Exception;
-use Google\Service\CustomSearchAPI;
-use Google\Service\CustomSearchAPI\Search;
-use Pimcore;
-use Pimcore\Cache;
-use Pimcore\Cache\RuntimeCache;
-use Pimcore\Google\Cse\Item;
-use Pimcore\Localization\LocaleServiceInterface;
-use Pimcore\Model;
-use Pimcore\Model\Paginator\PaginateListingInterface;
-
-class Cse implements PaginateListingInterface
```

Comment:
this requires a deprecation in 10.6 first

Reply:
nevermind, just saw https://github.com/pimcore/pimcore/issues/14279

Reply:
I removed all CSE parts.